download.go

```diff
@@ -200,6 +201,15 @@ func downloadImage(ctx context.Context) (context.Context, context.CancelFunc, er
 		return ctx, func() {}, err
 	}
 
+	if res.Header.Get("Content-Encoding") == "gzip" {
+		gzipBody, errGzip := gzip.NewReader(res.Body)
+		if errGzip != nil {
+			return ctx, func() {}, err
```

Comment:
```suggestion
			return ctx, func() {}, errGzip
```

Reply:
Good catch. I changed the name of the err variable to satisfy the linter.